### DISCUSSION TITLE
The url in the profile failed to open

### DISCUSSION THREAD

**Discussion by sy-records** [community/community#6739] [Mobile] [answered]
I filled in `lufei.dev` without the protocol `https://` @sy-records 

Failed to open when using mobile, I saw the error:

![IMG_6177](https://user-images.githubusercontent.com/33931153/138791761-516ca11f-cb53-4414-96a3-3c372f6ab473.jpg)

**Comment by mxie** [comment, marked as answer]
Thanks for letting us know! I'll relay this back to our engineers to investigate.

**Reply by aconsuegra** [reply]
We fixed this and the fix will be included in our upcoming update. Thanks for the report @sy-records!

**Reply by sy-records** [reply]
The problem persists in the latest version. `version 1.38.0 (50770266)`

**Reply by aconsuegra** [reply]
The fix is included in 1.39, which will be released next week

**Reply by umutcolakvermis75-oss** [reply]
Yes
